src/containers/StructurePage/plannedResource/AddExistingResource.tsx

```diff
@@ -234,7 +235,7 @@ const AddExistingResource = ({ onClose, resourceTypes, existingResourceIds, node
     if (!preview) return;
     let id = resourceId;
     if (!id) {
-      const isLearningpath = selectedType === RESOURCE_TYPE_LEARNING_PATH && "metaUrl" in preview;
+      const isLearningpath = selectedType?.id === RESOURCE_TYPE_LEARNING_PATH && "metaUrl" in preview;
```

Comment:
Dette må inn for at det skal gå an å legge til læringssti uten kontekst.
```suggestion
            const isLearningpath =
        selectedType?.id === RESOURCE_TYPE_LEARNING_PATH || ("metaUrl" in preview && preview.metaUrl);
```